core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceExceptionHandlerTest.java

```diff
@@ -277,6 +300,26 @@ public HttpResponse handler3(ServiceRequestContext ctx, HttpRequest req) {
         }
     }
 
+    @LoggingDecorator
+    @ExceptionHandler(CapturingExceptionHandler.class)
+    public static class MyService6 extends MyService1 {
```

Comment:
Should we remove `MyService1` from super class because the `requestConverterExceptionIsRelayed` tests methods only in `MyService6`?